Tests/UnitTests/Networking/Responses/Fixtures/PaywallData-Sample1.json

```diff
@@ -20,7 +20,7 @@
     "default_locale": "en_US",
     "config": {
         "packages": ["$rc_monthly", "$rc_annual"],
-        "header_image": "asset_name.png",
```

Comment:
@guido732 @lburdock as discussed in the doc, this is now an array. The current template only supports 1 but allows us to display multiple images in future templates.
I've added validation when decoding it to ensure that this is not empty. It would be nice if the frontend did the same to make sure we never save a template without an image.